src/benchmark.cc

```diff
@@ -52,56 +52,66 @@
 #include "thread_timer.h"
 
 // Print a list of benchmarks. This option overrides all other options.
-DEFINE_bool(benchmark_list_tests, false);
+DEFINE_bool(benchmark_list_tests,
+            benchmark::BoolFromEnv("benchmark_list_tests", false));
```

Comment:
This looks too intrusive, it should be handled on some other level of abstraction probably

Reply:
To be more specific, i think the e.g. `DEFINE_bool` itself should be modified.